### DISCUSSION TITLE
Unlisting this extension from the VS Code marketplace

### DISCUSSION THREAD

**Discussion by tristanlabelle** [vknabel/vscode-swift-development-environment#123] [General]
Newcomers to Swift in VS Code can get confused by the many Swift extensions available:

![image](https://github.com/vknabel/vscode-swift-development-environment/assets/2314287/ee0c03a4-dad3-4052-9b0d-f619931caa9a)

This extension was a great contribution to the early days of Swift in VS Code, but as stated in its readme, it is no longer being maintained or keeping up-to-date with recent Swift versions. I think the Swift community would benefit in rallying around the official Swift extension and avoid confusion, especially since this extension is named "maintained". I have previously been in touch with Martin Kase, who agreed to unlist his "Swift Language" extension and I was wondering if you'd be open to doing the same here? I would also follow up with Jin Mingjian with the same request.

**Comment by vknabel** [comment]
Great idea! To be honest I forgot about all the deprecation changes I wanted to perform, like putting up banners and officially deprecate it.
Thanks for the reminder. I'll take the extension down within the next few days.

**Reply by vknabel** [reply]
I just unpublished the extension and will archive this repo.
